component/Recipe/index.js

```diff
@@ -0,0 +1,83 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function Recipe({ recipe }) {
+  console.log(recipe);
+
+  return (
+    <StyledContainer>
+      <StyledImage
+        src={recipe.images.SMALL.url}
+        width={recipe.images.SMALL.width}
+        height={recipe.images.SMALL.height}
+        alt="Small Recipe"
+      />
+      <StyledHeading>{recipe.label}</StyledHeading>
+      <StyledLine />
+      <StyledParagraph>Time: {recipe.totalTime} mins</StyledParagraph>
+      <StyledAnchor href={recipe.url}>
+        {recipe.ingredients.length} Ingredients
+      </StyledAnchor>
+      <StyledAnchor href={recipe.url}>Instruction</StyledAnchor>
+    </StyledContainer>
+  );
+}
+
+const StyledGrid = styled.div`
```

Comment:
Please explain where you use that StyledGrid I don't get it :)

Reply:
Hi, I ended up not using it in the Recipe Component so I don’t necessarily need StyledGrid. I’ll remove it :)